PeerioTechnologies/peerio-mobile
# iOS: Ability to pick videos from the gallery

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/4901/ios-ability-to-pick-videos-from-the-gallery
#### Testing instructions  
1. When uploading a file in the files tab, I should be able to select a video
2. When choosing from gallery to share into a chat, I should be able to select a video

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
